main.py

```diff
@@ -112,6 +112,12 @@ def get(self):
 
 def get_redirect_route(regex_route, handler, defaults=None):
     """Returns a route that redirects /foo/ to /foo.
+    Args:
+        regex_route: str. For regex for routing
+        handler: function.  handler function
+        defaults: dict.
```

Comment:
Do you have any more information regarding `defaults`?

Reply:
done